topics/query-subscription-consumer.yaml

```diff
@@ -0,0 +1,9 @@
+topic: events-subscription-results
+description: Metric alert events consumer
+schemas:
+  - version: 1
+    compatibility_mode: none
+    type: json
+    resource: query-subscription-consumer.schema.json
```

Comment:
Also shouldn't be named after one of the clients. Recall that the producer will use this too when it is publishing the messages. You can call it `subscription-results.schema.json`